feature/security/gnsi/authz/tests/README.md

```diff
@@ -146,29 +146,29 @@ Prepare the following gRPC authorization policies.
       "name": "gnmi-get",
       "source": {
         "principals": [
-          "spiffe://test-abc.foo.bar/xyz/read-only",
+          "spiffe://test-abc.foo.bar/xyz/read-only"
         ]
       },
       "request": {
         "paths": ["/gnmi.GNMI/Get"]
       }
-    },
-  ],
+    }
+  ]
 }
 ```
 
 The following table describes policy `policy-normal-1`:
 
-Cert | gRIBI.Modify | gRIBI.Get | gNMI.Set | gRIBI.Get | gNOI.Time | gNOI.Ping | gNSI.Rotate | gNSI.Get
+Cert | gRIBI.Modify | gRIBI.Get | gNMI.Set | gNMI.Get | gNOI.Time | gNOI.Ping | gNSI.Rotate | gNSI.Get
 :--- | :---  | :--- | :---  | :---  | :---  | :--- | :--- | :-----
-cert_user_admin | allow | allow |allow |allow |allow |allow |allow |allow
+cert_user_admin | allow | allow |allow |allow |allow |allow |deny |deny
 cert_user_fake | deny |deny |deny |deny |deny |deny |deny |deny
 cert_gribi_modify | allow |allow |deny |deny |deny |deny |deny |deny
-cert_gnmi_set | deny |deny |deny |deny |deny |deny |allow |allow
-cert_gnoi_time |deny |deny |allow |allow |deny |deny |deny |deny
-cert_gnoi_ping |deny |deny |deny |deny |allow |deny |deny |deny
-cert_gnsi_probe |deny |deny |deny |deny |deny |allow |deny |deny
-cert_read_only |deny |deny |allow |allow |deny |deny |deny |allow
+cert_gnmi_set | deny |deny |allow |allow |deny |deny |deny |deny
+cert_gnoi_time |deny |deny |deny |deny |allow |deny |deny |deny
+cert_gnoi_ping |deny |deny |deny |deny |deny |allow |deny |deny
+cert_gnsi_probe |deny |deny |deny |deny |deny |deny |deny |deny
```

Comment:
Can you help add one more column about `gNSI.Probe` (and it will be `allowed` for the `cert_gnsi_probe`)? Thanks.

Reply:
Done